examples/resolver/resolver.go

```diff
@@ -0,0 +1,28 @@
+package resolver
+
+import (
+	"fmt"
+	"net/http"
+)
+
+//RequestURL simply returns
+//the request url from REQUEST_URI header
+//this should not be done in production applications
+type RequestURL struct {
```

Comment:
cool, but we should provide one or two resolvers that can be used out of the box in my opinon. Just to get ppl started quickly without implementing a lot